recipes/microhapulator/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.1.3" %}
+{% set sha256 = "d613d1c4ec977feab1f3ec7a53835b6b2bde9d48af2036c2b910a4d1d943641a" %}
+
+package:
+  name: microhapulator
+  version: '{{version}}'
+
+source:
+  url: https://github.com/bioforensics/microhapulator/archive/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  noarch: python
+  entry_points:
+    - mhpl8r = microhapulator.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - happer >=0.1
+    - insilicoseq >=1.3.6
+    - microhapdb >=0.3
+    - numpy >=1.15.4
+    - python >=3
+    - pyfaidx >=0.5.5.2
+    - pysam >=0.15.2
+
+test:
+  imports:
+    - microhapulator
+  requires:
+    - pytest >=3.10
+    - pytest-cov >=2.6
+    - wget
```

Comment:
This was generated I think. Do you need the requires? 

Is there some other useful test you can run, like a command? Nothing big, just something to test whether it looks like it should in principle work.